env0/data_git_token.go

```diff
@@ -0,0 +1,58 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataGitToken() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataGitTokenRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Description:  "the name of the git token",
+				Optional:     true,
+				ExactlyOneOf: []string{"name", "id"},
+			},
+			"id": {
+				Type:         schema.TypeString,
+				Description:  "id of the git token",
+				Optional:     true,
+				ExactlyOneOf: []string{"name", "id"},
+			},
+			"value": {
```

Comment:
If I'm not mistaken, API doesn't return this value anyway (or returns a masked value) so there is no value in having it in the data resource schema. 
The value of having this data resource in the first place is for users to query by name so they can re-use the git token's id in other templates, for example.

Reply:
@yaronya 
078cc22530fccc8e51a7043f84eb7b6883d15a96